tests/uptane_test.cc

```diff
@@ -153,9 +153,9 @@ TEST(uptane, sign) {
 }
 
 /*
- * \verify{\tst{153}} Check that aktualizr creates provisioning files if they
```

Comment:
Not a big deal, but could you get undo these whitespace changes? If they are caused by clang-format, let me know, but they seem undesirable.